core/store/src/db/mixeddb.rs

```diff
@@ -0,0 +1,141 @@
+use std::io;
+use std::sync::Arc;
+
+use crate::db::{DBIterator, DBSlice, DBTransaction, Database, StoreStatistics};
+use crate::DBCol;
+
+#[allow(dead_code)]
+#[derive(Clone, Copy, Debug)]
+pub enum ReadOrder {
+    ReadDBFirst,
```

Comment:
In this mode the results would be inconsistent in a [write, read] sequence for a key that exists in the read db.

As you suggeted, in the resharding tool we want WriteDBFirst. If there is no use for ReadDBFirst can we get rid of it to simplify things?

Reply:
Use is to change head without messing up the original DB. I use it like that in the FS tool.